gprofiler/profilers/java.py

```diff
@@ -465,14 +467,15 @@ def _get_start_cmd(self, interval: int, ap_timeout: int) -> List[str]:
             f"{self._get_ap_output_args()},interval={interval},"
             f"log={self._log_path_process}{',buildids' if self._buildids else ''}"
             f"{f',fdtransfer={self._fdtransfer_path}' if self._mode == 'cpu' else ''}"
-            f",safemode={self._ap_safemode},timeout={ap_timeout}{self._get_extra_ap_args()}"
+            f",safemode={self._ap_safemode},timeout={ap_timeout}"
+            f"{',lib' if self._insert_dso_name else ''}{self._get_extra_ap_args()}"
```

Comment:
Is it needed for both `start` and `stop`? (to add `lib`)

Reply:
Yes, the tests have revealed this.